cloudos_cli/__main__.py

```diff
@@ -1028,21 +1033,21 @@ def list_projects(apikey,
         print('\t' + str(cl) + '\n')
         print('\tSearching for projects in the following workspace: ' +
               f'{workspace_id}')
-    my_projects_r = cl.get_project_list(workspace_id, verify_ssl)
+    # Check if the user provided the --page option
+    ctx = click.get_current_context()
+    if ctx.get_parameter_source('page') == click.core.ParameterSource.DEFAULT:
+        get_all = True
+    else:
+        get_all = False
```

Comment:
```suggestion
    else:
        get_all = False
        if not isinstance(page, int) or page < 1:
            raise ValueError('Please, use a positive integer (>= 1) for the --page parameter')
```
Could we also add similar to cloudos job list? Thanks!